geometry_msgs/Pose left_controller_pose
geometry_msgs/Pose2D left_thumbstick_position
bool left_gripper
geometry_msgs/Pose right_controller_pose
geometry_msgs/Pose2D right_thumbstick_position
bool right_gripper